src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
+    total: row.total
```

Comment:
the total number of rows. I made that because it's the most straightforward way to do pagination. As far as returning the page and limit, I don't think it's necessary as the requestor know what they requested.
I favored limited impact over a more advanced API result.